doc/sphinx/addendum/micromega.rst

```diff
@@ -279,18 +322,31 @@ proof by abstracting monomials by variables.
    Goal forall x, -x^2 >= 0 -> x - 1 >= 0 -> False.
    intro x.
    psatz Z 2.
+   Qed.
```

Comment:
Maybe use `.. coqtop::` instead to show the output if it would be helpful to the reader

Reply:
Unfortunately, as the comment above says, this requires the external tool CSDP which will cause issues compiling the doc. Fortunately enough, the only output is the fact that the goal is proved, so not much is lost.